pkg/openstackprovisionserver/deployment.go

```diff
@@ -43,7 +43,7 @@ func Deployment(
 		// TODO might need tuning
 		TimeoutSeconds:      5,
 		PeriodSeconds:       3,
-		InitialDelaySeconds: 30,
+		InitialDelaySeconds: 60,
```

Comment:
Another option here that might allow us to avoid always adding 60 seconds is the startup probe's `failureThreshold` [1].  That way you can keep a low test interval but still give the pod lots of potential time to reach availability.  i.e.

```
  failureThreshold: 30
  periodSeconds: 10
```

That example would check every 10 seconds but only kill the pod if no successful response was received within 300 seconds.  I just wonder if this would be more flexible.

[1] https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes